gef-ui/server/controllers/application-submission/index.test.js

```diff
@@ -83,6 +83,17 @@ const MockFacilityResponse = () => {
   const res = {};
   res.status = 'IN_DEAL_STATUS';
   res.data = [];
+  res.items = [];
+  res.details = {
+    type: 'CASH',
+    name: 'Foundry4',
+    hasBeenIssued: true,
+    monthsOfCover: null,
+    coverStartDate: '2022-01-02T00:00:00.000+00:00',
+    shouldCoverStartOnSubmission: true,
+    changedToIssued: true,
+    coverEndDate: '2030-01-02T00:00:00.000+00:00',
+  };
```

Comment:
Looks like there is a mock file added with the same data - can we use that?

Reply:
done